src/Microsoft.Health.Fhir.Core.UnitTests/Features/Search/SearchValues/QuantitySearchValueTests.cs

```diff
@@ -81,6 +81,7 @@ public void GivenAStringContainingMoreThanTwoTokenSeparators_WhenParsing_ThenExc
         [InlineData(@"234.5|system|", "system", "", 234.5)]
         [InlineData(@"0|system|code", "system", "code", 0)]
         [InlineData(@"98.565656||code", "", "code", 98.565656)]
+        [InlineData(@"5.40e-3|system|code", "system", "code", 0.00540)]
```

Comment:
Could you also add a test case to the QuanitySearchTests/QuantitySearchTestFixture files.

Reply:
Done!